### PR TITLE
Add support for CVSS vectors display

### PR DESCRIPTION
**Pull request by ziadhany**
This is an example of how to display CVSS vectors

Screenshot 2023-09-29 at 03-09-53 VulnerableCode Vulnerability Details - VCID-115q-hx34-aaae

#1239

### CODE REVIEW

**Comment by ziadhany** [comment]
Should the table have horizontal or vertical heading? 

Do we need to display all CVSS vector info or just the Base Score Metrics? 

Should we display all options of attribute 
for ex: Attack Vector: Network, Adjacent Network, Local , Physical
or we should just display the value?

**Comment by ziadhany** [comment]
image

**Comment by TG1999** [comment]
@ziadhany please resolve conflicts

File: vulnerabilities/severity_systems.py [hunk 1]
```diff
@@ -49,6 +53,9 @@ def compute(self, scoring_elements: str) -> str:
         """
         return str(CVSS2(vector=scoring_elements).base_score)
 
+    def get(self, scoring_elements: str) -> dict:
```
Comment (TG1999) [inline vulnerabilities/severity_systems.py:56]: Please add tests for all the scenarios, also check if there may exist a case where it can fail.

**Review by TG1999 — CHANGES_REQUESTED** [review]
@ziadhany thanks++, review comments for your consideration

File: vulnerabilities/severity_systems.py (hunk 1 above)
Comment (ziadhany) [reply vulnerabilities/severity_systems.py:55]: Done

File: vulnerabilities/views.py [hunk 2]
```diff
@@ -130,6 +130,11 @@ def get_context_data(self, **kwargs):
                 "severity_score_range": get_severity_range(
                     {s.value for s in self.object.severities}
                 ),
+                "severity_vectors": [
+                    SCORING_SYSTEMS[s.scoring_system].get(s.scoring_elements)
```
[the remark this answers is not in the document]
Comment (ziadhany) [reply vulnerabilities/views.py]: Done

**Comment by TG1999** [comment]
@ziadhany please resolve merge conflicts : )

**Comment by ziadhany** [comment]
> @ziadhany please resolve merge conflicts : )

@TG1999 Done!

**Review by TG1999 — APPROVED** [review]
LGTM!

**Comment by TG1999** [comment]
@ziadhany please add changelog and merge this PR

**Review by TG1999 — CHANGES_REQUESTED** [review]
Please fix failing tests : )

**Comment by ziadhany** [comment]
> Please fix failing tests : )

Done

**Review by TG1999 — APPROVED** [review]
LGTM! thanks++

**Pull request merged**